### ISSUE TITLE
[UI] Bulk delete should exclude services bound to applications in Configuration page

### ISSUE THREAD

**Issue by mmartin24** [epinio/ui#161]

**Comment by richard-cox** [comment]
The selection is used to provide rows to all bulk action buttons. For this table though there is only one. When hovering over the delete button the user will be told it will only apply to some of the selection.

**Comment by mmartin24** [comment]
Yes, but it may result a bit confusing making the UX not so smooth to the end user. As an user, I think it makes sense that if an user should not able to perform an action, besides a message when hovering, it would nice the he/she is actually prevented from doing so. The message may not be the first thing to be read. I actually happened to me; I was focused in deleting the items and I did not realize about that message appearing during hovering.
